createrepo_c.spec

```diff
@@ -1,21 +1,8 @@
 %global libmodulemd_version 2.3.0
 
-%define __cmake_in_source_build 1
+%undefine __cmake_in_source_build
 
-# Bash completion (we need different approach for RHEL-6)
-%if 0%{?rhel} == 6
-%global bash_completion %config%{_sysconfdir}/bash_completion.d/createrepo_c.bash
-%else
 %global bash_completion %{_datadir}/bash-completion/completions/*
-%endif
-
-%if 0%{?rhel} && 0%{?rhel} <= 7
-%bcond_with python3
-%bcond_with drpm
-%else
-%bcond_without python3
-%bcond_without drpm
```

Comment:
The `drpm` switch is unrelated and still present in the spec file. Maybe we could remove it too, since we are effectively dropping RHEL 7 support with this? But you might just wanna keep it in for now, as the commits are about python.

Reply:
+1

Reply:
fixed